glaciers/anaconda-project.yml

```diff
@@ -19,15 +19,15 @@ channels:
 
 packages: &pkgs
 - python=3.11
-- bokeh >=3.3.4
-- notebook
-- datashader
-- holoviews
-- hvplot
-- panel >=1
+- bokeh>=3.4.0
+- notebook<7
+- datashader>=0.16.0
+- holoviews>=1.19
+- hvplot>0.9.2
+- panel>=1.4.1
 - spatialpandas
```

Comment:
Can you also add a min version for spatialpandas?

Reply:
Done